CMakeModules/SplitDebugInfo.cmake

```diff
@@ -0,0 +1,88 @@
+# Tailored after https://github.com/GerbilSoft/mcrecover/blob/master/cmake/macros/SplitDebugInformation.cmake
+# Minor modifications to original
+
+if (NOT MSVC)
+  include(CMakeFindBinUtils)
+  if (NOT APPLE AND NOT CMAKE_OBJCOPY)
```

Comment:
You can probably use find_program to find a different program on apple. I think there are g versions of equivalent programs available on brew.

Reply:
gnu binutils installed by brew can end in places where find_program doesn't look for by default. This happened on jesse(OSX system) too. Hence, I am using dsymutil on APPLE, which is guaranteed to be available from what I read so far.